api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/pvls/PregnantCalculation.java

```diff
@@ -171,7 +170,6 @@ private boolean isPregnantDueDate(Date lastVlDate, List<Obs> pregnantDueDateObsL
   private boolean isPregnantInProgram(Date lastVlDate, SimpleResult pregnantInProgramResut) {
 
     PatientState patientState = this.getPatientState(pregnantInProgramResut);
-
     return patientState != null
         && patientState.getStartDate() != null
         && patientState.getEndDate() == null
```

Comment:
What if the patientState.getEndDate() has value that falls within the reporting period or the dates ranges specified? will that be counted or we completely need this column to be null?

Reply:
@ningosi , now I'm considering patient Program.dateEnrolled to compare with the period range, I think using patientState.startDate could be the mistake in the pregnantCalculator

> Patients that are female and enrolled on PTV/ETC program during the period range.